streaming_main.py

```diff
@@ -511,6 +517,11 @@ def main() -> None:
         action="store_true",
         help="only parse the test files and fail/pass based on that",
     )
+    parser.add_argument(
+        "--ignore-regex-tests",
+        action="store_true",
+        help="ignore tests starting with test/built-ins/RegExp/ since they can take up to 80% of total runtime",
+    )
```

Comment:
Running the full test262 takes ~5 minutes with `-c 16` for me but without regex just over 1 minute so this was very useful and I thought it could also be useful to other people.

Reply:
How about something more generic like a negative filter? `--ignore <glob>` for instance.

Reply:
(Leaving aside ali's suggestion, which is a good idea imo) I don't see the point of this option at all, the only person who will be running a full test262 run is the CI runner, which obviously wont use this option, and people using test262 to check and iterate on their LibJS fixes will filter out the tests to only the ones who handle the specific feature they're working on (at least in my experience)

Reply:
I added the ignore flag.
What I noticed with this new runner is that it is so quick to run the test that I just run everything (apart from RegExp). 
But this way everyone can have their own workflow so that's probably best.

Reply:
Eh, it could also come in handy if you just don't want the results for a specific suite.